web-app/README.md

```diff
@@ -0,0 +1,27 @@
+# Faucet Web App
+
+This is a next.js web app that is used to interact with the faucet smart contract.
+We ask the user to connect using their wallet and a OAuth provider (GitHub or Discord) to protect against spam requests.
+
+## Environment Variables
+
+| Variables | Description |
+| -------- | ------- |
+| `NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID` | ProjectID use by WalletConnect connector https://cloud.walletconnect.com/ |
```

Comment:
Didn't know about this tool before, pretty interesting. Are you using email credentials to authorize, or connect a wallet that is deploying the smart contract (faucet contract in this case)? I think that would be great to use email / password credentials and link them to our [bitwarden](https://bitwarden.com/), so that multiple users can make changes to it of necessary.